<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePlatformDockWidget</class>
 <widget class="QDockWidget" name="CreatePlatformDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>284</height>
   </size>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Create Platform</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>100</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="2">
       <widget class="wkf::AltitudeComboBox" name="mAltitudeUnitsBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Heading</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="wkf::AngleComboBox" name="mHeadingUnitsBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mNameLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Side</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mSideLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="UtQtLatPosLineEdit" name="mLatitudeLineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="wkf::LocationSelector" name="mLocationButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="UtQtLonPosLineEdit" name="mLongitudeLineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="mSpeedLineEdit"/>
      </item>
      <item row="7" column="2">
       <widget class="wkf::SpeedComboBox" name="mSpeedUnitsBox"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="mHeadingLineEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="mAltitudeLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mCreatePlatformButton">
        <property name="text">
         <string>Create Platform</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtLatPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLonPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::LocationSelector</class>
   <extends>QPushButton</extends>
   <header>WkfSelectorWidgets.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::AltitudeComboBox</class>
   <extends>QComboBox</extends>
   <header>WkfUnitComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::AngleComboBox</class>
   <extends>QComboBox</extends>
   <header>WkfUnitComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>wkf::SpeedComboBox</class>
   <extends>QComboBox</extends>
   <header>WkfUnitComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mNameLineEdit</tabstop>
  <tabstop>mTypeComboBox</tabstop>
  <tabstop>mSideLineEdit</tabstop>
  <tabstop>mLatitudeLineEdit</tabstop>
  <tabstop>mLocationButton</tabstop>
  <tabstop>mLongitudeLineEdit</tabstop>
  <tabstop>mAltitudeLineEdit</tabstop>
  <tabstop>mAltitudeUnitsBox</tabstop>
  <tabstop>mHeadingLineEdit</tabstop>
  <tabstop>mHeadingUnitsBox</tabstop>
  <tabstop>mSpeedLineEdit</tabstop>
  <tabstop>mSpeedUnitsBox</tabstop>
  <tabstop>mCreatePlatformButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
